src/components/Calculator.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import '../index.css';
+
+const Calculator = () => {
+  const count = useState(0);
+
+  return (
+    <div className="calculator-flex">
+      <div className="output">{count}</div>
+      <div className="buttons">
+        <button type="button" className="button">AC</button>
+        <button type="button" className="button">+/-</button>
+        <button type="button" className="button">%</button>
+        <button type="button" className="button highlight">&divide;</button>
```

Comment:
- [ ] As my colleague said the Calculator.js should be a class-based app.

![mmm](https://user-images.githubusercontent.com/61562626/144114474-134f8a49-c3ea-4629-8534-8f833af43b9d.jpeg)